### DISCUSSION TITLE
逐火之蛾对外宣传部正式成立啦！

### DISCUSSION THREAD

**Discussion by azb-666** [share121/inter-knot#2687] [其他]
![凯文](https://github.com/user-attachments/assets/f41c8167-655f-4e31-ab53-0f171e29df2c)

接下来本部将会逐渐发布关于各位英桀以及现世各位女武神，崩坏兽，律者的秘密同人卡牌（其实就是自己制作的三国杀同人卡牌）！敬请期待我们逐火之蛾对外宣传部的下个情报！
情报预告：逐火英桀第一位，强大高冷的冰火领袖，凯文·卡斯兰娜！

**Comment by EDEN54418** [comment]
![Uploading QQ图片20240818151836.jpg…]()
